### ISSUE TITLE
VR Keyboard inputs dont send to mods. 

### ISSUE THREAD

**Issue by mathew2214** [jrbudda/Vivecraft_112#197]
using Vivecraft's keyboard with no text box active should just send the input to the game, allowing one to use arbitrary bindings from mods. this does not work. 
this makes it impossible to use mods like Morph. 
i am well aware of SteamVR input rebinding, and that helps workaround a few mods, but many mods dont show their inputs in that system.

**Comment by TheTrueForce** [comment]
This is actually an issue with those mods. They're checking the keyboard directly, bypassing Minecraft's keybind system.
You'll need to bug the mod authors about that. I'm pretty sure there's nothing vivecraft can do about them.

**Comment by Techjar** [comment]
> This is actually an issue with those mods. They're checking the keyboard directly, bypassing Minecraft's keybind system. 

Yeah that's basically what they're saying. They want the in-game keyboard to send inputs so they can use the key bindings from mods that refuse to use the vanilla key bindings system. We're against doing that though, as it's just dumb and mods shouldn't be reinventing the wheel.
